### ISSUE TITLE
Composite.execute cannot be called multiple times in the same mojit

### ISSUE THREAD

**Issue by aljimenez** [YahooArchive/mojito#1280]
The following use of composite.execute results in a crash since the Y.Parallel queue that is used internally is not cleared after execution is done.

``` javascript
ac.composite.execute(config, function (data, meta) {
    ac.composite.execute(newConfig, function (data2, meta2) {
        ...
    });
});
```

There are many valid use cases for this. In my case I would like to execute a mojit which then accumulates some data, which I then need to execute other mojits. One easy fix in composite.common.js would be to reset the Y.Parallel queue before calling the callback towards the end of the execute method. That way subsequent executions would use a new queue:

``` javascript
self.queue = new Y.Parallel({
    context: self
});

cb(content, meta);
```

**Comment by drewfish** [comment]
It seems reasonable to me to want to use ac.composite.execute() multiple times like you suggest.

**Comment by clarle** [comment]
Issue resolved in #1300, and released in 0.8.2. 

Thanks for the report @aljimenez!
